plugins/imjournal/imjournal.c

```diff
@@ -599,12 +618,28 @@ CODESTARTrunInput
 		skipOldMessages();
 	}
 
-	if (cs.bUseJnlPID) {
-		/* Use the PIDs obtained by journald. */
-		pid_field_name = "_PID";
-	} else {
-		/* Use the PIDs obtained from the message. */
-		pid_field_name = "SYSLOG_PID";
+	/* handling old "usepidfromsystem" option */
+	if (cs.bUseJnlPID != -1) {
+		free(cs.usePid);
+		cs.usePid = strdup("system");
+		errmsg.LogError(0, RS_RET_DEPRECATED,
```

Comment:
This is the place where I can cover new and old syntax with error message. Also I wanted to know if it possible to use parse_errmsg because I think that this code runs after config parsing. So line number will be inconsistent or not? @rgerhards